test/UnitTests/Microsoft.Testing.Platform.MSBuild.UnitTests/Microsoft.Testing.Platform.MSBuild.UnitTests.csproj

```diff
@@ -1,9 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(MicrosoftTestingTargetFrameworks);net462</TargetFrameworks>
+    <TargetFrameworks>net8.0;net9.0</TargetFrameworks>
```

Comment:
Maybe use `$(NetMinimum);$(NetCurrent)`? (I assume NetMinimum flows properly from Arcade.

Reply:
It would not work well when we move to arcade 10, we would only test `net8.0` and `net10.0`